training/src/test/scala/com/airbnb/aerosolve/training/FullRankLinearModelTest.scala

```diff
@@ -43,6 +43,11 @@ class FullRankLinearModelTest {
     testFullRankLinear("softmax", 0.8)
   }
 
+  @Test
+  def testFullRankLinearHinge() = {
+    testFullRankLinear("hinge", 0.8)
```

Comment:
They both get 100% correct just lowering it to make sure the test is stable. I will add a multi-label case later